modules/hosted-cluster-etcd-status.adoc

```diff
@@ -14,19 +14,24 @@ You can check the status of the etcd cluster health by logging into any etcd pod
 +
 [source,terminal]
 ----
-$ oc rsh -n <hosted_control_plane_namespace> -c etcd <etcd_pod_name>
+$ oc rsh -n openshift-etcd etcd <etcd_pod_name>
```

Comment:
Don't we need the `-c` here, assuming the `etcd` is a container?

Reply:
You're absolutely correct, thank you! Updated now. :slightly_smiling_face: